### PR TITLE
fix(headerbar): specify HeaderBar /api/me query fields required

### PR DESCRIPTION
**Pull request by mediremi**
Closes https://jira.dhis2.org/browse/LIBS-73

### CODE REVIEW

File: packages/widgets/src/HeaderBar/Profile.js [hunk 1]
```diff
@@ -49,34 +42,35 @@ export default class Profile extends React.Component {
...
             >
-                {this.userIcon(user, baseUrl)}
+                {this.renderUserIcon({ avatar, name, baseUrl })}
 
                 {this.state.show ? (
                     <ProfileMenu
-                        avatar={avatarPath(user.avatar, baseUrl)}
-                        name={user.name}
-                        email={user.email}
+                        avatarUrl={avatar ? avatarUrl(avatar, baseUrl) : null}
```
Comment (HendrikThePendric) [inline packages/widgets/src/HeaderBar/Profile.js:65]: I think I'd prefer to keep the complexity in the function instead of pulling it out.

**Review by HendrikThePendric — CHANGES_REQUESTED** [review]
This popped up in my lists of "review requested" list, but I haven't looked at this in great detail yet because there are 53 failing tests. The code is looking good. I did one minor suggestion. I'm happy to look at the failing tests with you tomorrow.

**Comment by mediremi** [comment]
Sorry about the failing tests I didn't realise cypress had network request fixtures. I've updated the URLs of `/api/me` requests in the cypress tests and now all tests pass.

**Review by HendrikThePendric — APPROVED** [review]
Excellent!

**Pull request merged**